source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -174,7 +173,7 @@ void AuthenticatorImpl::startVerify() {
   if (jwks_data_->getJwtProvider().clock_skew_seconds() > 0) {
     clock_skew_seconds = jwks_data_->getJwtProvider().clock_skew_seconds();
   }
-  status = jwt_->verifyTimeConstraint(absl::ToUnixSeconds(absl::Now()), clock_skew_seconds);
+  Status status = jwt_->verifyTimeConstraint(absl::ToUnixSeconds(absl::Now()), clock_skew_seconds);
```

Comment:
Add following code in line 239, right inside the function verifyKey()


```
   Status status;
   // check token cache, lookup function should verify expiration time
   if (jwks_data_->lookupTokenCache(jwt_->signature_, status)) {
       status = cached_status;
   } else {
       status =
      ::google::jwt_verify::verifyJwtWithoutTimeChecking(*jwt_, *jwks_data_->getJwksObj());	      
   }
```

Reply:
Added